.travis.yml

```diff
@@ -1,37 +1,50 @@
+    - secure: Fl7zB+DmZTER2j1PfHUlJORWCkqSWPViwqPeV+OvHK59Oisgv+MJ7w5jU+N5iEmKi7V6moDCpdnChAlLN4XxYNKWHhuw1jv1GAvr083qpTRikbHvzPGNGU1g3g5tResslYJRW+U8pbhl+i50I7i0+VbYshHDUvtKiC6Wfb4KSDkIGNPhekmw1RZ/tZkZPwgX95sAqioyeI/AGAlI7x209eIHL6igxt5auc4XsfqTU9fjIoyyp1jRP4WRJ3tatJshAiwM+9SI0vzZyIe7w3d6y9CtEGxcBlCcCYn18YJ1+7da3Bfxv4nrIfsRKuVW/p2cypc7N26WMgsVlUzXryX/Bqie4SvFBiEM+WJey8YW2wc8Fr0ndcFjPFIsvpo+uHQRkwzuGspbzmfIfOJVVKCbIhAuKQZSGuJB5bSKtQxk0Vgz35Q4ag8gr8UfrjjoMtP0bb4N7nzYTHB/osDDKT9gmM/lBWyjXPRtCHWDgPJxSWrLxoXVWnzGZC/6Jc0eeJQlVOZ3YUViqSV2x3cXlOjyzSCkoDGk7fMFgPL+YP1gvn47KMhN9MtrSY+vkGqwDWrXb+0Mq4z7EIog5ZD+qE5XB014lDnQEk7GVpp5DVktdMBXkCBxpmBPDnpPF2BuTgf7nAMtJcNgq0PpLYc2d4W9ZpB0rD5KO1G83/ai+KnCWig=
+
 jobs:
+  fast_finish: true
   include:
-  - stage: maven central release
-    jdk: openjdk8
-    script: skip
-    deploy:
-      provider: script
-      script: bash .travis.deploy.sh
-      skip_cleanup: true
-      on:
-        tags: true
-  - stage: github release
-    jdk: openjdk8
-    script: skip
-    deploy:
-      provider: releases
-      api_key:
-        secure: RvyJMne8BdDQGwiVFb4j6YQz1AVKDSbJNepHDNUt21lQSm4TGmJ61MKZdHI+UsCTlLTVPmDLM3SvFL6ILSkDVuM67VCSDRwBp+mmPqouMzFWPQVpfVjfPOjSrr3TTZON2h39RyIifdO7laMQ1nv3HuLCciEjtH7nQ35fHBbs5TkgNnBoUoVIeWmRO667RlJAGvGfFLjwi/a96BtyR3gBGvlosHCX1Utx4Tp8t6SHTrLKLxSDUTiQp63fuygJVOmlzEUKv94VGmwacft2QUoommK4vFkDji5v18+4RwfaN1py/5riO7FJMVhmsNMrD5FJBgaaD8C0ufXGWzDWZPwTeiYxH9D2Vug3ySyzwvwt37wEy9PdQaNipOzZvvcCsamA3P+nYuqoAchCIUoxwi3TrAzd/0t3n/VP8PzMvSbu/BxQg3HRo7yZJmUVJgkyExc/s35f2gKzzUsfO9We6NJCYA4CmSe1ftblMOplr4wTI+U3xV9hvcawhTgY1mENUv3zFbte/2rwai9GpEPxMh2AScxQ0b2Vu3t2O25zcEHGoMKziZ5TU83mbvnpTu5ImUfw0JEbWmZiaNgTkTjwMI7LsQSXZsmtiu8C2u7ohepiy2UPXht7FFm6vMhgRTry3gyh2GURRWUDM8pgctn7uYScmd5br6vdNlMIm4e5huj7Dak=
-      skip_cleanup: true
-      on:
-        tags: true
+    # OpenJDK
+    - stage: test
+      name: "OpenJDK 8"
+      jdk: openjdk8
+      script: mvn -q clean verify -B
+    - name: "OpenJDK 11"
+      if: type != pull_request
+      jdk: openjdk11
+      script: mvn -q clean verify -B
+    # GraalVM
+    - name: "GraalVM 20.0.0 (8)"
+      if: type != pull_request
+      os: linux
+      dist: xenial
+      install: . ./install-jdk.sh --url "https://github.com/graalvm/graalvm-ce-builds/releases/download/vm-20.0.0/graalvm-ce-java8-linux-amd64-20.0.0.tar.gz"
+      script: mvn -q clean verify -B
+    - name: "GraalVM 20.0.0 (11)"
+      if: type != pull_request
+      os: linux
+      dist: xenial
+      install: . ./install-jdk.sh --url "https://github.com/graalvm/graalvm-ce-builds/releases/download/vm-20.0.0/graalvm-ce-java11-linux-amd64-20.0.0.tar.gz"
+      script: mvn -q clean verify -B
+    - stage: deploy
+      name: Maven Central Release
+      if: type != pull_request
```

Comment:
IMO it is better to reverse the condition and be more precise when we want to deploy something (in case Travis adds more types, for example)

Reply:
Maybe the right condition here is just `branch = master`